docs/en_US/NNICTLDOC.md

```diff
@@ -567,6 +568,54 @@ Debug mode will disable version check function in Trialkeeper.
     nnictl package show
     ```
 
+<a name="feed"></a>
+![](https://placehold.it/15/1589F0/000000?text=+) `nnictl feed`
+
+* Description
+
+  You can use this command to feed several prior or supplementary trial hyperparameters and results for NNI hyperparameter tuning. The data are fed to the tuning algorithm (e.g., tuner or advisor).
+
+* Usage
+
+  ```bash
+  nnictl feed [OPTIONS]
+  ```
+* Options
+
+  |Name, shorthand|Required|Default|Description|
+  |------|------|------|------|
+  |id|  False| |The id of the experiment you want to feed data into|
+  |--file, -f|  True| |a file with data you want to feed in json format|
+
+* Details
+
+  The file with feed data must in json format, an example is shown below:
+
+  ```json
+  [
+    {"paramter": {"x": 0.5, "y": 0.9}, "value": 0.03},
+    {"paramter": {"x": 0.4, "y": 0.8}, "value": 0.05},
+    {"paramter": {"x": 0.3, "y": 0.7}, "value": 0.04}
+  ]
+  ```
+
+  Every element in the top level list is a sample. For our built-in tuners/advisors, each sample should have at least two keys: `parameter` and `value`. The `parameter` must match this experiment's search space, that is, all the keys (or hyperparameters) in `parameter` must match the keys in the search space. Otherwise, tuner/advisor may have unpredictable behavior. `Value` should follow the same rule of the input in `nni.report_final_result`, that is, either a number or a dict with a key named `default`. For your customized tuner/advisor, the file could have any json content depending on how you implement the corresponding methods (e.g., `feed_tuning_data`).
+
+  Currenctly, following tuner and advisor support feed data:
+
+  ```yml
+  builtinTunerName: TPE, Anneal, Evolution, SMAC, NetworkMorphism, MetisTuner
```

Comment:
Random?

Reply:
Supporting Random is unnecessary, so we don't support random and all Tuners that use random to generate parameters (such as Hyperband).